source/fe/fe_nothing.cc

```diff
@@ -41,6 +43,17 @@ FE_Nothing<dim, spacedim>::FE_Nothing(const unsigned int n_components,
 }
 
 
+
+template <int dim, int spacedim>
+FE_Nothing<dim, spacedim>::FE_Nothing(const unsigned int n_components,
+                                      const bool         dominate)
+  : FE_Nothing<dim, spacedim>(ReferenceCell::get_hypercube(dim),
```

Comment:
So by default, we assume the cell type is square or hex. Should we document this?

Reply:
I have updated the documentation.